pkg/hubble/metrics/api/context.go

```diff
@@ -32,6 +32,8 @@ const (
 	// ContextPodShort uses a short version of the pod name. It should
 	// typically map to the deployment/replicaset name
 	ContextPodShort
+	// ContextPodName
```

Comment:
Should we copy the documentation about `ContextPod` here and add a note about the namespace prefix in `ContextPod`?

Reply:
Ah yeah, I forgot to finish writing the doc comment.